tests/subworkflows/nf-core/annotation_ensemblvep/main.nf

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ANNOTATION_ENSEMBLVEP } from '../../../../subworkflows/nf-core/annotation_ensemblvep/main'
+
+workflow annotation_ensemblvep {
+    input = [
+        [ id:'test' ], // meta map
+        file(params.test_data['sarscov2']['illumina']['test_vcf'], checkIfExists: true)
+    ]
+
+    ANNOTATION_ENSEMBLVEP ( input, "WBcel235", "caenorhabditis_elegans", "104", [] )
```

Comment:
Need a test supplying every parameter too, no? I don't know the suggestions here on test coverage.

Reply:
Not sure what you meant here actually

Reply:
Aren't the test workflows supposed to cover all scenarios? 
So there should also be another test where the `[]` is replaced with a valid parameter.